pydantic/json.py

```diff
@@ -0,0 +1,40 @@
+import datetime
+from decimal import Decimal
+from enum import Enum
+from types import GeneratorType
+from uuid import UUID
+
+from .main import BaseModel
+
+__all__ = ['pydantic_encoder']
+
+
+def isoformat(o):
+    return o.isoformat()
+
+
+ENCODERS_BY_TYPE = {
+    UUID: str,
+    datetime.datetime: isoformat,
+    datetime.date: isoformat,
+    datetime.time: isoformat,
+    set: list,
+    frozenset: list,
+    GeneratorType: list,
+    bytes: lambda o: o.decode(),
+    Decimal: float,
+}
+
+
+def pydantic_encoder(obj):
+    if isinstance(obj, BaseModel):
+        return obj.dict()
```

Comment:
A Python newb question: Why not `dict(obj)` here?

Reply:
no difference, models are setup so `dict(obj) == obj.dict()`. But generally I use this way.